python/src/scipp/utils/dimensions.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+# @author Jan-Lukas Wynen
+
+
+def find_bin_edges(var, ds):
+    """
+    Returns a list of bin-edge dimension labels assuming that
+    var is a coord or attr of ds.
```

Comment:
If it's not you're going to get some weird results, should we check, or would it be safer to work fully off the `ds` (no var argument)?

Reply:
This function originally comes from the HTML formatter which passes individual Variables and Datasets around. So we would need to change those functions, too.

I think it is fine the way it is because it lets users also check Variables that are not actually stored in a Dataset.